opensearch-operator/api/v1/opensearch_types.go

```diff
@@ -85,11 +85,17 @@ type PVCSource struct {
 // ConfMgmt defines which additional services will be deployed
 type ConfMgmt struct {
 	AutoScaler  bool `json:"autoScaler,omitempty"`
-	Monitoring  bool `json:"monitoring,omitempty"`
 	VerUpdate   bool `json:"VerUpdate,omitempty"`
 	SmartScaler bool `json:"smartScaler,omitempty"`
 }
 
+type MonitoringStuck struct {
+	Enable         bool   `json:"enable,omitempty"`
+	MonitoringUser string `json:"monitoringUser,omitempty"`
```

Comment:
I'd prefer the user has the option to specify the name of a secret (via `corev1.LocalObjectReference`) that must contain username and password, and just default on the admin password secret if not specified.

Reply:
needs to implement that logic, maybe we can declare it as a phase 2 feature and add it in the future so we will be able to release tomorrow .